extension/background.js

```diff
@@ -1,7 +1,11 @@
-browser.experiments.normandy.setManualMode(true);
-
 browser.browserAction.onClicked.addListener(async () => {
   await browser.tabs.create({
     url: "content.html",
   });
 });
+
+if (DEVELOPMENT) {
+  browser.tabs.create({
+    url: "restore.html",
+  });
+}
```

Comment:
I wonder if we wanted finer grained control over this through an environment variable like:
```sh
NDT_RESTORE=last-url
```

i'm thinking about the future where i want to have it restore all the tabs you had opened

Reply:
But you know what I can deal with that in a future PR